### ISSUE TITLE
Audio Not Rendering in Bodymovin

### ISSUE THREAD

**Issue by beingthere** [airbnb/lottie-web#2854]
Hi I love having just discovered Lottie and Bodymovin. I'm trying to render an mp3 in AE Bodymovin (latest build) and followed the instructions of mr. Osborn from Lottie files on how to do it. The result I get is 'render failed'. I tried switching up the audio settings in both and any other setting that is associated with mp3 in Lottie with no success yet. It does say in Bodymovin that audio is experimental at this point. Has anyone had success and have any suggestions? Thank you.

**Comment by bodymovin** [comment]
hi, can you share the .aep?

**Comment by beingthere** [comment]
Yes and thank you so much for your fast reply. As an update - I was able to get a render of the shape layer in the comp but I couldn't get it to render the mp3 layer (for a .json file with audio included).  I think the previous file that failed to render anything was due to the fact that it was a download aep file from lottiefiles and I just added an mp3 layer to test audio rendering and in doing so I might have unintentionally 'broke' something in ae settings.    https://assets.codepen.io/2100127/test+sound+mouse+click.aep
